src/bodyElement.ts

```diff
@@ -82,18 +94,39 @@ interface MediaAtom {
     caption: Option<DocumentFragment>;
 }
 
-interface GuideAtom {
-    kind: ElementKind.GuideAtom;
+interface ExpandableAtom {
     html: string;
     title: string;
     id: string;
+    image?: string;
+    credit?: string;
+}
+
+interface GuideAtom extends ExpandableAtom {
+    kind: ElementKind.GuideAtom;
 }
 
-interface QandaAtom {
+interface QandaAtom extends ExpandableAtom {
     kind: ElementKind.QandaAtom;
-    html: string;
+}
+
+interface ProfileAtom extends ExpandableAtom {
+    kind: ElementKind.ProfileAtom;
+}
+
+interface TimelineItem {
```

Comment:
Do you reckon it would be possible to `import` this, and possibly some of the atom interfaces, from `atoms-rendering`?

Reply:
They do export that type so I'm using it now.